app/graphql/inputs.js

```diff
@@ -5,8 +5,8 @@ module.exports = gql`
     firstName: String!
     lastName: String!
     username: String!
-    email: String! @constraint(format: "email", minLength: 12, pattern: ".*@wolox.(ar|co|cl|com.ar)$")
-    password: String! @constraint(pattern: "^[a-zA-Z0-9_]*$", minLength: 8)
```

Comment:
Why do you remove this validations?

Reply:
because it presents problems in the scheme

Reply:
I added it again it works but it presents errors in the playground